geometry_msgs/Point target_point
float32 stop_distance
---
bool status
